web/src/features/feature-flags/FeatureFlagsManager.tsx

```diff
@@ -34,23 +39,31 @@ function Content({ features }: { features: FeatureFlags }) {
   });
 
   return (
-    <div>
+    <div className="flex flex-col">
       <p className="pb-1 font-poppins text-sm text-gray-600">Feature Flags</p>
-      {Object.entries(features).map(([key, value]) => (
-        <div className="flex w-full items-center justify-between text-sm" key={key}>
-          <label className="pr-8" htmlFor={key}>
-            {key}
-          </label>
-          <Switch.Root
-            className="relative h-[20px] w-[38px] cursor-default rounded-full bg-gray-300 outline-none data-[state=checked]:bg-brand-green"
-            id={key}
-            defaultChecked={Boolean(value)}
-            onCheckedChange={() => mutation.mutate(key)}
-          >
-            <Switch.Thumb className="block h-[16px] w-[16px] translate-x-0.5 rounded-full bg-white transition-transform duration-100 will-change-transform data-[state=checked]:translate-x-[19px]" />
-          </Switch.Root>
-        </div>
-      ))}
+      <div>
+        {Object.entries(features).map(([key, value]) => (
+          <div className="flex w-full items-center justify-between text-sm" key={key}>
+            <label className="pr-8" htmlFor={key}>
+              {key}
+            </label>
+            <Switch.Root
+              className="relative h-[20px] w-[38px] cursor-default rounded-full bg-gray-300 outline-none data-[state=checked]:bg-brand-green"
+              id={key}
+              defaultChecked={Boolean(value)}
+              onCheckedChange={() => mutation.mutate(key)}
+            >
+              <Switch.Thumb className="block h-[16px] w-[16px] translate-x-0.5 rounded-full bg-white transition-transform duration-100 will-change-transform data-[state=checked]:translate-x-[19px]" />
+            </Switch.Root>
+          </div>
+        ))}
+      </div>
+      <button
+        className="mt-2 self-end rounded  bg-green-900 p-1 text-xs text-white"
+        onClick={handleClearLocalStorage}
+      >
+        Clear Local Storage
+      </button>
```

Comment:
Can we maybe increase the text size a bit? It is very small

Reply:
🙌 bumped it up!